packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+import { DropdownMenu } from './DropdownMenu';
+import { DropdownPosition, DropdownDirection, DropdownContext } from './dropdownConstants';
+
+export interface DropdownProps extends React.HTMLProps<HTMLDivElement>{
+  /** Anything which can be rendered in a dropdown */
+  children?: React.ReactNode; 
+  /** Classes applied to root element of dropdown */
+  className?: string; 
+  /** Array of DropdownItem nodes that will be rendered in the dropdown Menu list */
+  dropdownItems?: any[]; 
+  /** Flag to indicate if menu is opened */
+  isOpen?: boolean; 
+  /** Display the toggle with no border or background */
+  isPlain?: boolean; 
+  /** Indicates where menu will be aligned horizontally */
+  position?: DropdownPosition | 'right' | 'left';
+  /** Display menu above or below dropdown toggle */
+  direction?: DropdownDirection | 'up' | 'down';
+  /** Flag to indicate if dropdown has groups */
+  isGrouped?: boolean;
+  /** Placeholder to use custom toggle elements */
+  toggle: React.ReactElement<any>; 
```

Comment:
Is the toggle optional? I ask because the description indicates the prop is a 'placeholder'.

Reply:
The toggle is required, but I agree that the comment is poorly worded. I tried to reword it. :)